### ISSUE TITLE
getting t is not a function

### ISSUE THREAD

**Issue by da1z** [undecaf/zbar-wasm#9]
I am using it like this: 
```TS
import { createCanvas, loadImage } from 'canvas';
import { scanImageData } from '@undecaf/zbar-wasm';

const getImageData = async (src) => {
  const img = await loadImage(src);
  const canvas = createCanvas(img.width, img.height);
  const ctx = canvas.getContext('2d');
  ctx.drawImage(img, 0, 0);
  return ctx.getImageData(0, 0, img.width, img.height);
};

const url = 'src/IMG_3361.jpg';
const main = async () => {
  const img = await getImageData(url);
  const res = await scanImageData(img);
  console.log(res[0]?.typeName);
  console.log(res[0]);
  console.log(res[0]?.decode());
};

main();
```
and getting

`   TypeError: t is not a function at n (file:///Users/Projects/scna/node_modules/@undecaf/zbar-wasm/dist/main.js:15:706)`

**Comment by undecaf** [comment]
While I am looking into this, you could try to replace
```javascript
import { scanImageData } from '@undecaf/zbar-wasm';
```
with 
```javascript
import { createRequire } from 'module';
const require = createRequire(import.meta.url);
const { scanImageData } = require('@undecaf/zbar-wasm');
```
as a workaround. This is equivalent to what the missing function `t` should do.

**Comment by undecaf** [comment]
@da1z : Should be fixed in version 0.9.16, please feel free to re-open.
